bftengine/src/bftengine/ValidationOnlyIdentityManager.hpp

```diff
@@ -0,0 +1,41 @@
+// Concord
+//
+// Copyright (c) 2020-2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+#include "SigManager.hpp"
+#pragma once
+
+namespace bftEngine::impl {
+/* This class is a hack to enable the validation of replica signatures
+   using fixed keys without initializing a CryptoManager object.
+   Messages such as CheckpointMsg use a global singleton to expose their validation logic,
```

Comment:
The verify method eventually calls to `verifyNonReplicaSig`. Maybe I'm missing something, but checkpoint messages are signed by replicas, right?

Reply:
Indeed. As I explained in a previous discussion, the naming is poor, but the meaning is that Sigmanager's internal map is used instead of redirecting to CryptoManager.

Reply:
renamed.